packages/flutter_tools/lib/src/drive/web_driver_service.dart

```diff
@@ -111,6 +111,8 @@ class WebDriverService extends DriverService {
         'Unable to start WebDriver Session for Flutter for Web testing. \n'
         'Make sure you have the correct WebDriver Server running at $driverPort. \n'
         'Make sure the WebDriver Server matches option --browser-name. \n'
+        'For more information see: '
+        'https://flutter.dev/docs/testing/integration-tests#running-in-a-browser \n'
```

Comment:
the trailing space before the newline here is what i was referring to

Reply:
Okay, thanks for pointing it out. I assumed this was the correct style based on the existing message, but let me know if you prefer otherwise and I can remove the trailing spaces for the entire message.

Reply:
Friendly ping @Hixie

Reply:
The existing text also needs fixing, I hadn't noticed that.

There shouldn't be a trailing space before a newline.

Reply:
I removed all of the trailing spaces before new lines in this message 👍